corehq/util/htmx_action.py

```diff
@@ -49,14 +49,20 @@ def make_edit(request, *args, **kwargs):
     Example docs here: commcarehq.org/styleguide/b5/htmx_alpine/
     See working demo here: commcarehq.org/styleguide/demo/htmx_todo/
     """
-    default_htmx_error_template = "prototype/htmx/partials/htmx_action_error.html"
+    default_htmx_error_template = ""
```

Comment:
> This correctly adds a default error template
Where does that happen?  It looks like this is blank by default now

Reply:
agh I forgot to cherry-pick that one